packages/@o3r/third-party/src/bridge/ab-testing/ab-testing-bridge.ts

```diff
@@ -0,0 +1,108 @@
+import {
+  BehaviorSubject, distinctUntilChanged, firstValueFrom, Observable
+} from 'rxjs';
+import {Logger} from '@o3r/core';
+
+export interface AbTestBridgeInterface<T> {
+  /**
+   * start an AB testing experiment
+   */
+  start(exp: T): void | Promise<void>;
+  /**
+   * stop an AB testing experiment
+   */
+  stop(exp?: T): void | Promise<void>;
+}
+
+export interface AbTestBridgeConfig {
+  /**
+   * Reference to communicate with the bridge from the window
+   * @default 'abTestBridge'
+   */
+  bridgeName: string;
+  /**
+   * Debug logger
+   * @default console
+   */
+  logger: Logger;
+  /**
+   * Event a third party can subscribe to before starting the communication with the bridge
+   * @default 'ab-test-ready'
+   */
+  readyEventName: string;
+}
+
+/**
+ * Default options that will represent the interface
+ */
+const defaultOptions: AbTestBridgeConfig = {
+  bridgeName: 'abTestBridge',
+  readyEventName: 'ab-test-ready',
+  logger: console
+};
+
+export class AbTestBridge<T> implements AbTestBridgeInterface<T> {
+  /**
+   * Behaviour subject to control the experiments via the exposed interface
+   */
+  private experimentSubject$: BehaviorSubject<T[]> = new BehaviorSubject<T[]>([]);
+  /**
+   * Options to configure the communication between the AB Testing bridge and third parties
+   */
+  private options: AbTestBridgeConfig;
+  /**
+   * Observable with the list of AB testing experiments currently applied
+   */
+  public experiments$: Observable<T[]>;
+
+  constructor(private equalityModel: (value1?: T, value2?: T) => boolean, options?: Partial<AbTestBridgeConfig>) {
+    this.experiments$ = this.experimentSubject$.asObservable().pipe(
+      distinctUntilChanged((experimentsA: T[], experimentsB: T[]) =>
+        experimentsB.length === experimentsA.length && experimentsA.every((eA) => experimentsB.find((eB) => equalityModel(eA, eB))
+        ))
+    );
+    this.options = {
+      ...defaultOptions,
+      ...options
+    };
+    if (!(window as any)[this.options.bridgeName]) {
+      // eslint-disable-next-line @typescript-eslint/dot-notation, dot-notation
+      (window as any)[this.options.bridgeName] = {start: this.start.bind(this), stop: this.stop.bind(this)};
+    }
+    document.dispatchEvent(new CustomEvent(this.options.readyEventName));
+  }
+
+  /**
+   * Use configured logger to log AB testing related information
+   * @param args
+   */
+  private log(...args: any[]) {
+    this.options.logger.log('A/B Test', ...args);
```

Comment:
Sounds like `debug` message actually.

```suggestion
    (this.options.logger.debug || this.options.logger.log)('A/B Test', ...args);
```